tests/matrix_free/fe_remote_evaluation_02.with_p4est=true.mpirun=1.output

```diff
@@ -0,0 +1,21 @@
+
+DEAL:0::Setup MatrixFree...
+DEAL:0::
+DEAL:0::Test the access of remote gradients at points in face batches...
+DEAL:0::  Setup Communicator...
+DEAL:0::  Test with first_selected_component = 0
+DEAL:0::  OK
```

Comment:
@jh66637 Only print one "OK"  and no headers.

Reply:
OK, the tests already passed with this version. I think this is finally working :partying_face: 

Anyway, the tests now only print one OK and I squashed all commits. Thank you for the help :heart: